README.md

```diff
@@ -79,15 +79,5 @@ docker run --env-file .env todo_app
 test container run using CI pipeline in My-CI-Pipeline.yml file everytime you open or reopen a pull request or push to repository
 can also be observed on GitHub workflows to see which ones pass and fail.
 
-## deploying and running on heroku
-firstly login to the heroku using
-heroku container:login 
-build the image using 
-docker build --tag registry.heroku.com/corndelm8/web --target production .
-push to heroku using 
-docker push registry.heroku.com/corndelm8/web
-release using
-heroku container:release web
-open up in web browser using
-heroku open or the link http://corndelm8.herokuapp.com/
+
```

Comment:
Well remembered to clean this up, though it would be nice to link to your new Azure website now!